packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx

```diff
@@ -30,17 +98,40 @@ This component requires the following peer dependencies be installed in your app
 | react-dom | ^16.8.5 |
 | react-intl | ^2.8.0 |
 
-<!-- AUTO-GENERATED-CONTENT:END -->
+## Properties
+
+### FolderTree Props Table
+<FolderTreePropsTable />
+
+### FolderTreeItem Props Table
+<FolderTreeItemPropsTable />
```

Comment:
Same comment as above, this time with subfolderItems prop.  If possible replace FolderTree.Items in the Description field with "items" and maybe reword the first sentence to be something like "List of items to display in a subfolder when it is clicked."

Reply:
Updated: [18cf213](https://github.com/cerner/terra-framework/pull/1973/commits/18cf213eb5eabfc53af77417b4a42ba34b3520f7)